src/components/Header/Header.tsx

```diff
@@ -50,16 +53,114 @@ const HomepageLink = styled(Link)`
   }
 `
 
-const jobLinkMobile = css`
-  ${from.desktop} {
-    display: none;
-  }
+const socialLinksWrapper = css`
+  display: flex;
+  left: 0;
+  position: absolute;
+  top: ${space[4]}px;
 `
 
+const socialLinkWrapper = css`
+  margin-left: ${space[2]}px;
+`
+
+const socialButton = {
+  button: {
+    textPrimary: colors.primaryText,
+    backgroundPrimary: colors.silverGrey
+  }
+}
+
+// TODO: Add this to Source?
+const SvgRSS = () => {
```

Comment:
I extracted SvgRSS path from the figma design https://www.figma.com/proto/Ozt3bSnuO0i07D7IPjo4lC/theguardian.engineering?scaling=min-zoom&node-id=1%3A7

Is it something we will like to move into the Source? Happy to do that, if so

cc
@SiAdcock